client/src/app/shared/components/list-view-table/list-view-table.component.ts

```diff
@@ -165,6 +165,35 @@ export class ListViewTableComponent<V extends BaseViewModel, M extends BaseModel
     @Input()
     public showFilterBar = true;
 
+    /**
+     * Fix value for the height of the rows in the virtual-scroll-list.
+     */
+    @Input()
+    public vScrollFixed = 110;
+
+    /**
+     * Option to apply additional classes to the virtual-scrolling-list.
+     */
+    @Input()
+    public set classes(values: { [key: string]: boolean }) {
+        this._classes = values;
+    }
+
+    /**
+     * Returns the list of classes, that are applied to the virtual-scrolling-list.
+     * Already prepared for the `[ngClass]`-property.
+     *
+     * @returns An object looking like `{ [key: string]: boolean }`.
+     */
+    public get classes(): { [key: string]: boolean } {
+        const defaultClasses = {
+            'virtual-scroll-with-head-bar ngrid-hide-head': this.showFilterBar,
+            'virtual-scroll-full-page': !this.showFilterBar,
+            multiselect: this.multiSelect
+        };
+        return Object.assign(this._classes, defaultClasses);
```

Comment:
Won't the `Object.assign` will overwrite some custom set values for the keys of `defaultClasses`? If this is wanted, please add the keys to `set classes` with the hint, that they will be overwritten.